### DISCUSSION TITLE
How to change widget size?

### DISCUSSION THREAD

**Discussion by Roy-Kid** [manzt/anywidget#562] [General]
Hi everyone,

I have embedded my react project in any widget. But I have a problem I can not fix it. My widget is quite small so my canvas can not display. 
![image](https://github.com/manzt/anywidget/assets/42854324/8f77720e-3d62-4a15-892b-3cb470a5e9e5)
My component is initialized successfully, and I can find the canvas in debug tools, but the outside container is too small to show the canvas. 
Can someone help me to style the widget? I am super fresh to frontend

**Comment by manzt** [comment]
Are you able to share any code or a repo?

**Reply by Roy-Kid** [reply]
Sorry for replying late.

This is my project, and I try my best to make it a minimal reproduction environment. 
https://github.com/MolCrafts/molvis/blob/master/src/molvis/src/ts/widget.tsx
I use babylon.js as my render engine and related code is located in `src/view`. I want to implement UI by using react and the code is in `src/page`. My plan is to separately develop the view and page, bundle them to esm then use it in any widget. I try to directly import `molvis` from `view`, it works(means initialization is successful and <canvas> is created), but not display successfully...
